scripts/qiita-load-qebil-downloads

```diff
@@ -0,0 +1,211 @@
+                added = True
+                files_used.append(f)
+                break
+        if not added:
+            warnings.append(f'Not supported: {f}')
+
+    if not preps:
+        warnings.append('No valid preparations found')
+
+    for dt, ptfps in preps.items():
+        dt = dt.replace('_', ' ')
+        print(f'==> Processing {dt}')
+        for ptfp in ptfps:
+            print(f'   {ptfp}')
+            files_used.append(ptfp)
+            pt = load_prep_template_from_cmd(ptfp, study.id, dt)
+            pt.ebi_experiment_accessions = pt.get_category(
+                'experiment_accession')
+            pt.ebi_run_accessions = pt.get_category('run_accession')
+
+            library_layout = set(pt.get_category('library_layout').values())
+
+            run_prefixes = pt.get_category('run_prefix').values()
+
+            if len(run_prefixes) != len(set(run_prefixes)):
+                warnings.append(
+                    f'Run prefixes are not unique; prep-id: {pt.id}')
+                continue
+
+            filepaths = []
+            for rp in run_prefixes:
+                matches = sorted([f for f in files if rp in f])
+                if library_layout == {'PAIRED'}:
+                    if len(matches) != 2:
+                        warnings.append(f"{pt.id}: {rp} doesn't match PAIRED "
+                                        "library layout")
+                        continue
+                    filepaths.append((matches[0], 1))
+                    filepaths.append((matches[1], 2))
+                elif library_layout == {'SINGLE'}:
+                    if len(matches) != 1:
+                        warnings.append(f"{pt.id}: {rp} doesn't match SINGLE "
+                                        "library layout")
+                        continue
+                    filepaths.append((matches[0], 1))
+                else:
+                    warnings.append('Unknown library layout: '
+                                    f'{library_layout}; prep-id: {pt.id}')
+            files_used.extend([x for x, _ in filepaths])
+
+            lfp = len(filepaths)
+            lrp = len(run_prefixes)
+            if library_layout == {'PAIRED'} and lfp != lrp*2:
+                warnings.append('Not a valid number of files/run_prefixes '
+                                f'({lfp}/{lrp}) for "PAIRED"; prep-id: '
+                                f'{pt.id}')
+                continue
+            elif library_layout == {'SINGLE'} and lfp != lrp:
+                warnings.append('Not a valid number of files/run_prefixes '
+                                f'({lfp}/{lrp}) for "SINGLE"; prep-id: '
+                                f'{pt.id}')
+                continue
+
+            artifact = Artifact.create(filepaths, 'per_sample_FASTQ',
+                                       prep_template=pt, move_files=False)
+    notes = ''
+    if warnings:
+        notes = '<b>Warnings</b>:<ol>%s</ol>\n' % ''.join(
```

Comment:
html formatting here feeeeels like it should be done at the visualization layer on template render, but I would understand why doing so here is pragmatic